README.md

```diff
@@ -1,39 +1,126 @@
+![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/Azure.azure-iot-sdk-python)
+
+This repository contains code for the Azure IoT SDKs for Python.  This enables python developers to easily create IoT devices that work semealessly 
 
 *If you're looking for the v1.x.x client library, it is now preserved in the [v1-deprecated](https://github.com/Azure/azure-iot-sdk-python/tree/v1-deprecated) branch.*
 
 **Note that these SDKs are currently in preview, and are subject to change.**
 
-# SDKs
+## Python SDK
 
-This repository contains the following SDKs:
+This repository contains the following:
 
-* [Azure IoT Device SDK](azure-iot-device) - /azure-iot-device
-    * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
-    * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
-    * Handle direct methods invoked by the Azure IoT hub on a device
-    * Handle twin events and report twin updates
-    * *Still in development*
-        - *Blob/File upload*
-        - *Invoking method from a module client onto a leaf device*
+* *Azure IoT Device SDK*
 
-* Azure IoT Hub SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Hub
+  * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
+  * Handle direct methods invoked by the Azure IoT hub on a device
+  * Handle twin events and report twin updates
+  * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
+  * *Still in development*
+    * *Blob/File upload*
+    * *Invoking method from a module client onto a leaf device*
 
-* Azure IoT Hub Provisioning SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Device Provisioning Service
+* Azure IoT Hub SDK
+  * Do service/management operations on the Azure IoT Hub
+  * *Still in development*
+    * C2D messaging
+    * Direct Methods
+    * Configurations
+    * Query devices
+    * IoTHub Job
 
-# How to install the SDKs
+  * Do service/management operations on the Azure IoT Device Provisioning Service
 
-```
+## How to install the SDKs
+
+The SDK is contained in various Pip Packages that can be installed with the following commands:
+
+For the Device SDK the following command:
+
+```Shell
 pip install azure-iot-device
 ```
 
-# Contributing
+For the IoTHub SDK the following command:
+
+```Shell
+pip install azure-iot-hub
+```
+
+## Python SDKs Feature
+
+:heavy_check_mark: feature available  :heavy_multiplication_x: feature planned but not yet supported  :heavy_minus_sign: no support planned*
+
+*Features that are not planned may be prioritized in a future release, but are not currently in our 6 short term plans
+
+### Device Client Library
```

Comment:
> Device Client Library [](start = 3, length = 22)

I would suggest adding the package name in brackets, with a link to the that part of the repo

i.e. `Device Client Library (azure-iot-device)`